src/scripts/active-inactive-orgunit/index.js

```diff
@@ -98,69 +94,70 @@ OrganisationUnitActivator.prototype.processDataElementGroup = function (dataElem
         );
 
         //Process every dataElements
-        dataElements.forEach(function (dataElement) {
-            dataElement.dataElementGroup = dataElementGroup;
+        dataElements.forEach(function (dataElement) { 
             console.info("\nConfig:\n", JSON.stringify(dataElement, null, "\t"));
-            _this.prepareDataElement(dataElement);
-            _this.processDataElements(dataElement);
+            var isDataElementValid = _this.prepareDataElement(dataElement);
+            if (isDataElementValid) {
+                _this.processDataElements(dataElement, dataValues);
+            }
         });
         _this.asyncCalls--;
     });
 };
 
 /**
  * Builds the dataElement attributes
+@param dataElement The active dataElement
  */
 OrganisationUnitActivator.prototype.prepareDataElement = function (dataElement) {
-
-    var _this = this;
+    var attributeUids = this.attributeUids;
```

Comment:
Maybe to be coherent with the rest of the code follow the _this=this approach instead of creating a variable. What do u think?